### PR TITLE
Idva5 693 implement frontend backend connection

### PR DESCRIPTION
**Pull request by swalke-ch**
few changes required for integration with frontend

https://companieshouse.atlassian.net/browse/IDVA5-693

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -7,7 +7,7 @@ locals {
   container_port            = "18642" # default Java port to match start script
   docker_repo               = "acsp-api"
   lb_listener_rule_priority = 18
-  lb_listener_paths         = ["/acsp-api/*"]
+  lb_listener_paths         = ["/acsp-api/*", "/transactions/*/acsp*", "/private/transactions/*/accounts*"]
```
Comment (sampankumar) [inline terraform/groups/ecs-service/locals.tf:10]: private url seems wrong, I believe that should also have acsp at the end.

**Review by atedros-ch — APPROVED** [review]
LGTM

File: terraform/groups/ecs-service/locals.tf (hunk 1 above)
Comment (swalke-ch) [reply terraform/groups/ecs-service/locals.tf]: sorted it in separate pr

**Review by sampankumar — APPROVED** [review]

**Review by abhiahirv1 — APPROVED** [review]

**Pull request merged**